### PR TITLE
Presentation: Test app improvements

### PR DESCRIPTION
**Pull request by saskliutas**
Added IPC handler with functions to delete/update elements. This allows to setup test cases for hierarchy/content updates easier.

### CODE REVIEW

**Comment by saskliutas** [comment]
@grigasp Do you think it would be worth to initialize `presentation-backend` in `ReadWrite` mode by default always or maybe when running in electron? Or leave it to be changed when testing and by default always be in `Read` mode?

File: test-apps/presentation-test-app/src/frontend/api/MyAppFrontend.ts [hunk 1]
```diff
@@ -45,4 +55,35 @@ export class MyAppFrontend {
...
+    if (!IpcApp.isValid)
+      return;
+    return this.callIpc("updateElement", imodel.key, newProps);
+  }
+
+  public static async deleteElements(imodel: IModelConnection, elementIds: Id64Arg) {
+    if (!IpcApp.isValid)
+      return;
+    return this.callIpc("deleteElements", imodel.key, elementIds);
+  }
```
Comment (grigasp) [inline test-apps/presentation-test-app/src/frontend/api/MyAppFrontend.ts:69]: IMO these two should throw when it's not an IPC app. That would make it very clear why they don't work.

**Review by grigasp — COMMENTED** [review]
> @grigasp Do you think it would be worth to initialize `presentation-backend` in `ReadWrite` mode by default always or maybe when running in electron? Or leave it to be changed when testing and by default always be in `Read` mode?

Let's always initialize with `ReadWrite` - that'll make us include all the additional work we need to do for supporting read-write case, even when in web app mode. Might highlight some read-write specific issues...

File: test-apps/presentation-test-app/src/frontend/api/MyAppFrontend.ts (hunk 1 above)
Comment (saskliutas) [reply test-apps/presentation-test-app/src/frontend/api/MyAppFrontend.ts]: Changed

**Review by grigasp — APPROVED** [review]

**Pull request merged**